synapse/handlers/presence.py

```diff
@@ -491,9 +501,12 @@ async def set_state(self, target_user, state, ignore_status_msg=False):
         if not self.hs.config.use_presence:
             return
 
-        # Proxy request to master
+        # Proxy request to instance that writes presence
         await self._set_state_client(
-            user_id=user_id, state=state, ignore_status_msg=ignore_status_msg
+            instance_name=self._presence_writer_instance,
```

Comment:
TIL that the thing returned by `ReplicationEndpoint.make_client` takes an `instance_name` param. Its docstring totally doesn't say that :-S.